app/models/event.rb

```diff
@@ -24,7 +24,6 @@ class Event < ApplicationRecord
 
   LOCAL_EVENTS = [
     'Event::DraftDecision',
-    'Event::NewVersion',
```

Comment:
Are we sure we want to delete this?

Reply:
That's not what `LOCAL_EVENTS` signifies. LOCAL_EVENTS is used to work out what the app store thinks the most recent updated_at date is, by knowing which events are synced to the app store.

Reply:
That makes sense, worth adding a comment?